ts/parts/Stacking.ts

```diff
@@ -535,13 +536,24 @@ Axis.prototype.buildStacks = function (this: Highcharts.Axis): void {
     var axisSeries = this.series,
         reversedStacks = pick(this.options.reversedStacks, true),
         len = axisSeries.length,
+        actualSeries,
+        options,
         i;
 
     if (!this.isXAxis) {
         this.usePercentage = false;
         i = len;
         while (i--) {
-            axisSeries[reversedStacks ? i : len - i - 1].setStackedPoints();
+            actualSeries = axisSeries[reversedStacks ? i : len - i - 1];
+            options = actualSeries.options;
+
+            // Threshold should always be set to 0 in case of percent
+            // stacking in inverted polar bars
```

Comment:
I think that setting `threshold` with some other value than 0 in case when `stacking` is set to `percent` doesn't make much sense in general. I got rid of this piece of code. Should we let the user set a custom `threshold` value and make it affect percent stacking (that's how it works right now on cartesian plots)? Chart will work but the data won't be correctly placed.

Reply:
Yes I think we should go for consistence. So if this is a bug with polar, then it is also a bug with cartesian.